services/platform/platform.go

```diff
@@ -156,7 +156,9 @@ func (s *Service) SignOut(ctx context.Context) error {
 
 	_, err = api.NewAuthAPIClient(cc).SignOut(ctx, &api.SignOutRequest{})
 	if err != nil {
-		if st, ok := status.FromError(err); !ok || st.Code() != codes.InvalidArgument {
+		// If SaaS credentials have become invalid then go ahead with the log out instead of returning error.
+		// For more details see this: https://jira.percona.com/browse/PMM-7965
+		if st, ok := status.FromError(err); !ok || st.Code() != codes.InvalidArgument && st.Message() != "Invalid credentials." {
```

Comment:
Maybe it makes sense to log out user on any action if his credentials are bad?

Reply:
i don't understand, this condition will proceed with the logout in every case if it gets `codes.Unauthenticated`.

Reply:
Nevermind, I don't know why, but I thought that there are other methods.